src/App.css

```diff
@@ -18,6 +18,12 @@
   font-size: large;
 }
 
+h2 {
+  margin-top: 75px;
+  margin-bottom: -75px;
+  color: #95a5a6;
+}
+
 @keyframes App-logo-spin {
```

Comment:
That was the base React app logo, it's a good point though to bear in mind. For now I removed it as it's not necessary